samples/README.md

```diff
@@ -32,6 +34,40 @@ Before running the samples, make sure you've followed the steps outlined in
 
 
 
+### Create_cluster
+
+View the [source code](https://github.com/googleapis/nodejs-cloud-container/blob/main/samples/create_cluster.js).
+
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-cloud-container&page=editor&open_in_editor=samples/create_cluster.js,samples/README.md)
+
+__Usage:__
+
+
+`node samples/create_cluster.js`
```

Comment:
This autogenerated usage is not correct. Im not sure how I can change this. In any case it doesn't break anything. The user will get a message indicating required args on first run